Web/Edubase.Web.UI/Areas/Groups/Views/Group/EditDetails.cshtml

```diff
@@ -137,8 +137,12 @@
                             @Html.TextBoxFor(x => x.GroupId, new { @class = "govuk-input", maxlength = 120 })
                         </div>
                     }
+                    else
+                    {
+                        @Html.HiddenFor(x => x.GroupId)
+                    }
```

Comment:
Was this broken previously? 

It seems logical to have it present, but noting that it wasn't there before and I don't see it removed in the diff I wonder if it is required?

Reply:
I think its because its not included for L.SecureSingleAcademyTrust so would not be in the form.